test/unittest/render-commands.py.ts

```diff
@@ -316,7 +316,7 @@ describe('renderCommandsPYTest', () => {
         });
         const oriFile = path.join(
             `${__dirname}`,
-            '../../test/unittest/expected/generated/ori_commands.py',
```

Comment:
why do we change this?

Reply:
it use a same filename with other testcase, causing issue occasionally

Reply:
I see